sound/soc/sof/intel/hda.h

```diff
@@ -159,6 +159,7 @@
 
 #define HDA_DSP_STREAM_RESET_TIMEOUT		300
 #define HDA_DSP_CL_TRIGGER_TIMEOUT		300
+#define HDA_DSP_STREAM_TRIGGER_TIMEOUT		300
```

Comment:
is this the same as legacy or Skylake, if yes add a comment. This looks like copy-paste and not an intentional definition

Reply:
In the Skylake, it is hard code.  OK, I will add a comment to it.